info.yml

```diff
@@ -15,6 +15,11 @@ configuration:
         default_value: Scene Breakdown
         description: Specify a name for the app, its' menu item and the ui.
 
+    app_display_mode:
+        type: str
```

Comment:
Have you had time to consider changing this to a boolean as @barbara-darkshot recommended?

https://github.com/shotgunsoftware/tk-multi-breakdown2/pull/42#discussion_r1098457125

Just being complete before closing #42 as well - thanks!